components/LegacyAnimation/LegacyAnimation.styled.js

```diff
@@ -72,8 +53,8 @@ export const Blob = styled.div`
   height: 100%;
   border-radius: 100%;
   position: absolute;
-  mix-blend-mode: multiply;
+  /* mix-blend-mode: multiply; */
```

Comment:
```suggestion
```

Reply:
should stay in for now as we might use it later (work in progress :P)